tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1217,6 +1217,76 @@ TEST_F(OnnxImporterTest, importConvTransposeOutputShape) {
   convTransposeTestHelper(filename, expectedDims, expectedValues);
 }
 
+/// Test loading Range with int32 datatype.
+TEST(onnx, importRangeInt32) {
```

Comment:
This and `importRangeInt64` are essentially the exact same. Can we just add a single templatized function call to call into for these two tests?

Reply:
Done